patches/third_party/devtools-frontend/src/front_end-ui-legacy-themeColors.css.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/front_end/ui/legacy/themeColors.css b/front_end/ui/legacy/themeColors.css
-index 4f3ea9873fc04c7127430c91bc60844fcdd23af6..d7785abeda946b3651ec01805e64962cda860fc7 100644
---- a/front_end/ui/legacy/themeColors.css
-+++ b/front_end/ui/legacy/themeColors.css
-@@ -592,7 +592,7 @@
-   /* Outlines */
- 
-   --sys-color-outline: var(--ref-palette-neutral-variant60);
--  --sys-color-tonal-outline: var(--ref-palette-secondary50);
-+  --sys-color-tonal-outline: var(--sys-color-primary);
```

Comment:
Any reason why this patch went away?

Reply:
Looks like that filename was renamed in devtools-frontend here https://github.com/ChromeDevTools/devtools-frontend/commit/91816c4c3389eb4b923f4f1a17e90481d3bfdd70. 

Fixed here https://github.com/brave/brave-core/pull/27172/commits/1f0bdaab109105441b89295c44c41a6c59d7e055. Thanks!

cc: @cdesouza-chromium